docs/designs/publish.md

```diff
@@ -0,0 +1,43 @@
+# Publish
+
+## Publish Manifest File
+
+`docfx` generates a manifest file for publishing at `{output-path}/.publish.json` after build. It is the entry point for publishing. An example `.publish.json` looks like this:
+
+```javascript
+{
+    "publish": [
+        {
+            "url": "/dotnet/api/system.string",
+            "path": "dotnet/api/system.string.json",
+            "hash": "d41d8cd98f00b204e9800998ecf8427e",
+            "locale": "en-us",
+            "monikers": ["netstandard-2.0"],
+
+            // additional properties needed for publish
+        },
+        {
+            "url": "/dotnet/media/logo.png",
+            "path": "dotnet/media/logo.png",
+            "hash": "d41d8cd98f00b204e9800998ecf8427e",
+            "locale": "en-us",
+            "monikers": ["netstandard-2.0", "netstandard-2.1"],
+
+            // additional properties needed for publish
+        }
```

Comment:
should `.xremap.json`, `.filemap.json` be added here too?

Reply:
no, system generated files have well known names, they are not added here.